### PR TITLE
RH content from custom CDN

### PR DESCRIPTION
**Pull request by Lennonka**
FYI, this is for disconnected environments.

* [x] I am familiar with the [contributing](https://github.com/theforeman/foreman-documentation/blob/master/CONTRIBUTING.md) guidelines.

Please cherry-pick my commits into:

* [x] Foreman 3.4/Katello 4.6
* [x] Foreman 3.3/Katello 4.5
* [ ] Foreman 3.2/Katello 4.4
* [ ] Foreman 3.1/Katello 4.3

### CODE REVIEW

File: guides/common/modules/proc_configuring-server-to-consume-content-from-a-custom-cdn.adoc [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+For more information, see {ContentManagementDocURL}Importing_Custom_SSL_Certificates_content-management[Importing Custom SSL Certificates] in _{ContentManagementDocTitle}_.
+* You have uploaded a manifest to your organization.
+
+.Procedure
+. In the {ProjectWebUI}, navigate to *Content* > *Subscriptions*.
+. Click the *Manage Manifest* button.
+. Select the *CDN Configuration* tab.
+. Select the *Custom CDN* tab.
+. In the *URL* field, enter the URL of your CDN server from which you want {ProjectServer} to consume Red{nbsp}Hat repositories.
+. Optional: In the *SSL CA Content Credential*, select the SSL certificate of the CDN server.
```
Comment (parthaa) [inline guides/common/modules/proc_configuring-server-to-consume-content-from-a-custom-cdn.adoc:22]: You might want to mention that the CA Certificate needs to be added to the sat as a content credential (this dropdown only shows content credentials.) May be that is obvious. Leave it to you :)

File: guides/common/modules/proc_configuring-server-to-consume-content-from-a-custom-cdn.adoc [hunk 2]
```diff
@@ -0,0 +1,39 @@
+[id="configuring-{project-context}-server-to-consume-content-from-a-custom-cdn_{context}"]
+= Configuring {ProjectServerTitle} to Consume Content from a Custom CDN
+
+If you have an internal Content Delivery Network (CDN) server, you can configure your {ProjectServer} to consume Red{nbsp}Hat repositories from this CDN server instead of the Red{nbsp}Hat CDN.
```
Comment (parthaa) [inline guides/common/modules/proc_configuring-server-to-consume-content-from-a-custom-cdn.adoc:4]: May be
```If you have an internal Content Delivery Network  or are serving content via an accessible web server ...```

File: guides/common/modules/proc_configuring-server-to-consume-content-from-a-custom-cdn.adoc (hunk 1 above)
Comment (parthaa) [reply guides/common/modules/proc_configuring-server-to-consume-content-from-a-custom-cdn.adoc:22]: Wait its in the prereq. N/M

Comment (Lennonka) [reply guides/common/modules/proc_configuring-server-to-consume-content-from-a-custom-cdn.adoc:22]: @parthaa That's it in Prerequisites :)

File: guides/common/modules/proc_configuring-server-to-consume-content-from-a-custom-cdn.adoc [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+A CDN server can be any web server that mirrors repositories in the same directory structure as the Red{nbsp}Hat CDN.
```
Comment (Lennonka) [inline guides/common/modules/proc_configuring-server-to-consume-content-from-a-custom-cdn.adoc:5]: @parthaa And this is fine?

**Review by maximiliankolb — APPROVED** [review]

**Pull request merged**